enterprise/static/enterprise/enterprise_course_enrollment_page.js

```diff
@@ -0,0 +1,60 @@
+function focusableElementsSelector() {
+    return [
+        'a[href], area[href], input:not([disabled]), select:not([disabled]),',
+        'textarea:not([disabled]), button:not([disabled]), iframe, object, embed,',
+        '*[tabindex], *[contenteditable]'
+    ].join(' ');
+}
+function enableTabIndexingOn(containerSelector) {
+    var container = $(containerSelector),
+        focusableItems = container.find('*').filter(focusableElementsSelector()).filter(':visible');
+    container.attr('aria-hidden', 'false');
+    focusableItems.attr('tabindex', '0');
+}
+function disableTabIndexingOn(containerSelector) {
+    var container = $(containerSelector),
+    focusableItems = container.find('*').filter(focusableElementsSelector()).filter(':visible');
+    container.attr('aria-hidden', 'true');
+    focusableItems.attr('tabindex', '-1');
+}
+function showModal(modalSelector) {
+    $(modalSelector).attr('aria-hidden', 'false');
+    $(modalSelector).show();
+    $("#modal-close-button").focus();
+    $('body').addClass('open-modal');
+    disableTabIndexingOn('.enterprise-container');
+}
+function hideModal(modalSelector) {
+    $(modalSelector).attr('aria-hidden', 'true');
+    $(modalSelector).hide();
+    $('body').removeClass('open-modal');
+    enableTabIndexingOn('.enterprise-container');
+    $("#view-course-details-link").focus();
+}
```

Comment:
@replaceafill Similar comment here: You could DRY out `showModal` and `hideModal` using the approach described above (introduce a `toggleModal` function and use value of `newState` parameter to determine behavior).